content/_code-samples/build-a-wallet/js/6_domain-verification.js

```diff
@@ -0,0 +1,152 @@
+const { app, BrowserWindow, ipcMain } = require('electron')
+const path = require('path')
+const xrpl = require("xrpl")
+const { sendXrp } = require('./library/5_helpers')
+const { verify } = require('./library/6_helpers')
+
+const testnetUrl = "wss://s.altnet.rippletest.net:51233"
+
+let reserveBaseXrp = null
+let reserveIncrementXrp = null
+let client = null
+let wallet = null
+
+const fixedAddress = 'rn95xwUymaMyzAKnZUGuynjZ6qk9RzV4Q7'
+const fixedSeed = 'sEdSMHvzzWCLSuh2BiTgQihfyARVAKw'
+
+const prepareAccountData = (rawAccountData) => {
+    const numOwners = rawAccountData.OwnerCount || 0
+
+    let xrpReserve = null
+    if (reserveBaseXrp && reserveIncrementXrp) {
+        //TODO: Decimal?
```

Comment:
Not quite sure what this TODO means, just flagging it because it's in each file.

Reply:
Taken care of in an update